app/views/contact_us/contacts/new.html.erb

```diff
@@ -1,111 +1,95 @@
-<% javascript "contacts/new_contact.js" %>
+<% javascript "views/contacts/new_contact.js" %>
 
 <h1>
   <%= _("Contact Us") %>
 </h1>
 <p>
-  <%= raw _('%{application_name} is provided by the %{organisation_name}. You can find out more about us on our <a href="%{organisation_url}" target="_blank">website</a>. If you would like to contact us about %{application_name}, please fill out the form below.') % {organisation_name: Rails.configuration.branding[:organisation][:name],
+  <%= raw _('%{application_name} is provided by the %{organisation_name}.<br /> You can find out more about us on our <a href="%{organisation_url}" target="_blank">website</a>. If you would like to contact us about %{application_name}, please fill out the form below.') % {organisation_name: Rails.configuration.branding[:organisation][:name],
```

Comment:
@briri pootle turns red and warns the translator when they miss out brackets, tags, quotes in the translated string as compared to original - so that's a first line of defense. Second line is we can see the diff before we merge the new translation in ;)